packages/react-components/react-tabster/src/hooks/useModalAttributes.ts

```diff
@@ -5,10 +5,19 @@ import { useTabster } from './useTabster';
 
 export interface UseModalAttributesOptions {
   /**
-   * Traps focus inside the elements the attributes are applied
+   * Traps focus inside the elements the attributes are applied.
+   * It is recommended not to use the `absolute` variant since
```

Comment:
This comment is out of date now. What do you think about something like

> Traps focus inside the elements the attributes are applied to.
> Prefer this to `legacyTrapFocus`.

Reply:
```suggestion
   * Prefer this to `legacyTrapFocus`
```